docs/prow.md

```diff
@@ -0,0 +1,397 @@
+  - [Periodic Jobs](#periodic-jobs)
+    - [Image Updater Tooling](#image-updater-tooling)
+    - [Resource Group Cleanup](#resource-group-cleanup)
+    - [Periodic E2E Tests](#periodic-e2e-tests)
+- [EV2 Pipeline Integration](#ev2-pipeline-integration)
+- [Working with Prow Jobs](#working-with-prow-jobs)
+- [Best Practices](#best-practices)
+- [Troubleshooting](#troubleshooting)
+- [Related Documentation](#related-documentation)
+
+## Overview
+
+Prow is a Kubernetes-based CI/CD system originally developed for Kubernetes itself and now used across the OpenShift ecosystem. ARO HCP's Prow jobs are managed in the OpenShift release repository, separate from this codebase, which allows for centralized CI infrastructure management.
+
+**Prow Dashboard:** [https://prow.ci.openshift.org/?repo=Azure%2FARO-HCP](https://prow.ci.openshift.org/?repo=Azure%2FARO-HCP)
+
+The jobs are organized into two main categories:
+
+- **Presubmit jobs**: Run on pull requests to validate changes before merging
+- **Periodic jobs**: Run on a schedule to perform routine testing and maintenance
+
+## Job Categories
+
+### Presubmit Jobs
+
+Presubmit jobs run automatically or on-demand for pull requests to the main branch. These jobs validate code changes before they are merged.
+
+#### Quick Reference
+
+| Job | Trigger | Always Runs | Required | Environment |
+|-----|---------|-------------|----------|-------------|
+| images | `/test images` | Yes | Yes | - |
+| image-updater-images | `/test image-updater-images` | Yes | Yes | - |
+| periodic-images | `/test periodic-images` | Yes | Yes | - |
+| frontend-simulation | `/test frontend-simulation` | Yes | Yes | - |
+| e2e-parallel | `/test e2e-parallel` | Yes | No | Dev (westus3) |
+| integration-e2e-parallel | `/test integration-e2e-parallel` | No | No | Int (uksouth) |
+| stage-e2e-parallel | `/test stage-e2e-parallel` | No | No | Stage (uksouth) |
+| prod-e2e-parallel | `/test prod-e2e-parallel` | No | No | Prod (uksouth) |
+
+#### Images
+
+| Property | Value |
+|----------|-------|
+| **Job Names** | [`pull-ci-Azure-ARO-HCP-main-images`](https://prow.ci.openshift.org/?job=pull-ci-Azure-ARO-HCP-main-images)<br>[`pull-ci-Azure-ARO-HCP-main-image-updater-images`](https://prow.ci.openshift.org/?job=pull-ci-Azure-ARO-HCP-main-image-updater-images)<br>[`pull-ci-Azure-ARO-HCP-main-periodic-images`](https://prow.ci.openshift.org/?job=pull-ci-Azure-ARO-HCP-main-periodic-images) |
+| **Triggers** | `/test images`<br>`/test image-updater-images`<br>`/test periodic-images` |
+| **Status** | Always runs (required) |
+| **Step Registry** | [images](https://steps.ci.openshift.org/job?org=Azure&repo=ARO-HCP&branch=main&test=images)<br>[image-updater-images](https://steps.ci.openshift.org/job?org=Azure&repo=ARO-HCP&branch=main&test=image-updater-images)<br>[periodic-images](https://steps.ci.openshift.org/job?org=Azure&repo=ARO-HCP&branch=main&test=periodic-images) |
```

Comment:
These links are broken.

Reply:
Removed in https://github.com/Azure/ARO-HCP/pull/3424/commits/71481d15768e635c9d17f3e4c3fe8eda3505b94f